testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -55,31 +57,47 @@ do_one_repo () {
   cd "$WORKSPACE/testeng-ci"
   pip install -r requirements/base.txt
 
-  # if present and non-empty, set the PR description from .git/cleanup-python-code-message
-  # this will allow jobs using this script to make custom PR descriptions
-  if [ -s "$repo_dir/.git/cleanup-python-code-message" ]
+    # if present and non-empty, get additional information for PR body from .git/cleanup-python-code-description
+  if [ -s "$repo_dir/.git/cleanup-python-code-description" ]
   then
-     local message=$(cat "$repo_dir/.git/cleanup-python-code-message")
+    local pr_description_from_file="$(cat "$repo_dir/.git/cleanup-python-code-description")"
   else
-     local message="$(cat <<EOF
+    local pr_description_from_file=''
+  fi
+
+  local description="$(cat <<EOF
+$base_pr_body
+
+<details>
+<summary> Additional information from script execution</summary>
+$pr_description_from_file
```

Comment:
```suggestion
<summary>Additional information from script execution</summary>

$pr_description_from_file
```

nit: I'd suggest an additional line break here after the summary element just to avoid an annoying Markdown situation where if description from file starts with a code block, it needs an empty line before it. (Otherwise it doesn't render as a code block.)

Suggestion also removes stray space.